src/components/Chips/__stories__/chips.stories.mdx

```diff
@@ -84,6 +84,15 @@ Chips are compact elements that represent an input, attribute, or action. They m
   </Story>
 </Canvas>
 
+### With avatars
+
+<Canvas>
+  <Story name="With avatars">
```

Comment:
yay!!!

Reply:
let's add also a snapshot test for chip with avatar? same structure as the chip with icon tests if exist